runtime/flang/curdir.c

```diff
@@ -115,8 +115,12 @@ void __fort_gethostname(host) char *host;
     }
     p = un.nodename;
   }
-  strcpy(host, p);
 #else
-  strcpy(host, "localhost");
-#endif	  
+  #include <Winsock2.h>
+  s = gethostname(&p, 256);
```

Comment:
this needs `Ws2_32.lib` right?

Reply:
Yeah there are few libraries that are included. I'll toss that in.